core/templates/dev/head/css/oppia.css

```diff
@@ -2322,10 +2335,46 @@ div#ng-curtain {
   margin-bottom: 10px;
 }
 
+.oppia-notifications-tray {
+  margin-left: -8px;
+}
+
 .oppia-notifications-dashboard-card {
   padding: 15px 40px;
 }
 
+.oppia-notification-type {
+  float: left;
+  margin-right: 5px;
+}
+
+.oppia-notification-last-updated {
+  float: right;
+}
+
+.clearfix:after {
+  content: ".";
+  clear: both;
+  display: block;
+  height: 0;
+  visibility: hidden;
+}
+
+.oppia-notification-author-username,
+.oppia-notification-subject {
+  float: left;
```

Comment:
I'm actually a bit surprised that this is needed. Couldn't you just use a span element (to keep it inline) instead of a div?